### ISSUE TITLE
does this mod work with offline/ cracked players too?

### ISSUE THREAD

**Issue by simploita78** [intergrav/vital#2]
i have a Minecraft premium account but some of my friends doesnt i just want to ask that if this modpack work with my friend's cracked account (Tlauncher) if yes, thank you and sorry for inconvenience, if not then Can you please make it work. simply because whats the point of playing **Alone**. 

I hope you understand 
i appreciate your work

One more thing...

 Does this while playing with this mod, suppose i am playing on 1.20.4 and my friend on 1.20.6. Now ? can he joion my world ?

**Comment by Thinkseal** [comment]
you should test also they do not need the modpack to join your world

**Comment by worldwidepixel** [comment]
Seeing as the Vital project is a Modpack, it comes down to how the mods installed behave with an offline account. While this project does not endorse its usage with offline/cracked accounts, I don't see any reason why it wouldn't work.

TLDR [try it and see](https://tryitands.ee/)

**Comment by simploita78** [comment]
> you should test also they do not need the modpack to join your world

ok ill do it thanks

**Comment by simploita78** [comment]
> Seeing as the Vital project is a Modpack, it comes down to how the mods installed behave with an offline account. While this project does not endorse its usage with offline/cracked accounts, I don't see any reason why it wouldn't work.
> 
> TLDR [try it and see](https://tryitands.ee/)

Ok and, Thank you for taking the time to respond
